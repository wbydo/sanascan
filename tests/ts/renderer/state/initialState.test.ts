import { initialState } from "./initialState";

test("initialState", () => {
  expect(initialState).toMatchSnapshot();
});
