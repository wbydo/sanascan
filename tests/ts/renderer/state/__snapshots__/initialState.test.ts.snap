// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initialState 1`] = `
Object {
  "configWindow": Object {
    "isActive": false,
    "scanSpeed": 0,
  },
  "cursol": Object {
    "activeColumn": 0,
    "activeRow": 0,
    "direction": "column",
    "mode": "proposal",
  },
  "developerMode": Object {
    "estimator": false,
    "isActive": true,
  },
  "estimator": Object {
    "id": null,
    "result": "",
  },
  "timer": Object {
    "isActive": false,
    "scanSpeed": 500,
  },
}
`;
